packages/web-twig/src/Resources/components/Dropdown/README.md

```diff
@@ -0,0 +1,72 @@
+# Dropdown
+
+This is the Twig implementation of the [Dropdown] component.
+
+Basic example usage:
+
+```twig
+<Button data-toggle="dropdown" data-target="DropdownExample">Open Dropdown</Button>
+<Dropdown id="DropdownExample">Dropdown Content</Dropdown>
```

Comment:
**⚠️ `DropdownWrapper` component (which carries the `.DropdownWrapper` class) needs to be created yet.**

`DropdownWrapper` should be in all examples that present the complete composition because it ensures that `Dropdown` is positioned correctly.

See [`TooltipWrapper`](https://github.com/lmc-eu/spirit-design-system/tree/main/packages/web-twig/src/Resources/components/Tooltip#linking-with-content) to get the idea.

Reply:
👍 Done.